src/SpringrollGame.js

```diff
@@ -50,38 +47,35 @@ class SpringrollGame {
                 plugins: {
                     // FactoryPlugin is not necessary for Springroll, however it demonstrates
                     // how to setup and install a Phaser.Plugin.
-                    global: [ { key: "FactoryPlugin", plugin: FactoryPlugin, start: true } ]
+                    global: [{ key: "FactoryPlugin", plugin: FactoryPlugin, start: true }]
                 }
             });
 
-            // Listen for when the game is booted.
-            this.game.events.once("boot", () => {
-                // Create a Springroll.SafeScaleManager.
-                this.safeScale = new SafeScaleManager({
-                    width: GAMEPLAY.WIDTH,
-                    height: GAMEPLAY.HEIGHT,
-                    safeWidth: GAMEPLAY.SAFE_WIDTH,
-                    safeHeight: GAMEPLAY.SAFE_HEIGHT,
-                    callback: this.onWindowResize.bind(this)
-                });
-            })
+            // Create a Springroll.SafeScaleManager.
```

Comment:
Note: removed the boot event listener because it wasn't working and isn't needed for this to function anyway